lib/compress/zstd_compress.c

```diff
@@ -1469,56 +1520,57 @@ static U32 ZSTD_dictAndWindowLog(U32 windowLog, U64 srcSize, U64 dictSize)
  * `mode` is the mode for parameter adjustment. See docs for `ZSTD_CParamMode_e`.
  *  note : `srcSize==0` means 0!
  *  condition : cPar is presumed validated (can be checked using ZSTD_checkCParams()). */
-static ZSTD_compressionParameters
-ZSTD_adjustCParams_internal(ZSTD_compressionParameters cPar,
+static void
+ZSTD_adjustCParams_internal(ZSTD_CCtx_params* params,
```

Comment:
Could benefit from a rename: `ZSTD_adjustCCtxParams_internal()`.
In turn, the new name would help in extending the scope of this function beyond the original `ZSTD_compressionParameters` fields.